dvc/fs/local.py

```diff
@@ -32,7 +32,9 @@ def open(path_info, mode="r", encoding=None, **kwargs):
 
     def exists(self, path_info) -> bool:
         assert isinstance(path_info, str) or path_info.scheme == "local"
-        return self.fs.exists(path_info)
+        # NOTE: os.fspath could be removed after fsspec 2021.06.1
+        # https://github.com/intake/filesystem_spec/issues/666
+        return self.fs.exists(os.fspath(path_info))
```

Comment:
Do we need to use `fsspec` here? I think this is the only place where we use it.

Reply:
@skshetry The plan is to slowly start using fsspec's localfs instead of ours. So it is not necessary right now, sure.